### PR TITLE
#20 Access Customer User accounts via LDAP

### PR DESCRIPTION
**Pull request by kusid**
- Documentation updates.

### CODE REVIEW

File: mkdocs/docs/admin-guide.md [hunk 1]
```diff
@@ -269,17 +271,17 @@ result: 0 Success
 ```
   
 ## How to find Herd-MDL logs in CloudWatch
-Note: Logs are created in cloudwatch only when the log content is not empty, therefore, some of the log streams mentioned below may not be found if the logs are empty.
+Note: Logs are created in cloudwatch only when the log content is NOT empty, therefore; some of the log streams mentioned below may not be found if there is no content in the logs.
```
Comment (rongwang0930) [inline mkdocs/docs/admin-guide.md]: therefore,

File: mkdocs/docs/authorizations-overview.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+RBAC, Authorizations and Security in MDL
```
Comment (rongwang0930) [inline mkdocs/docs/authorizations-overview.md]: RBAC, what is this?

File: mkdocs/docs/user-guide.md [hunk 3]
```diff
@@ -126,25 +126,27 @@ Load data using the Uploader Tool, an ingestion utility created by the Herd team
...
+  ]
 }
 ```
 
 *   Create a manifest file named manifest.json with the contents above in the directory with the jar
 *   In that same directory, execute the following command (your uploader version will differ)
 
 ```
-java -jar herd-uploader-0.71.0.jar -l ./data -m ./manifest.json -V -H {HERD\_LOAD\_BALANCER\_DNS\_NAME}
+java -jar herd-uploader-0.71.0.jar -l ./data -m ./manifest.json -V -H {HERD_LOAD_BALANCER_DNS_NAME}
```
Comment (rongwang0930) [inline mkdocs/docs/user-guide.md]: I think this uploader version doesn't work

File: mkdocs/docs/admin-guide.md (hunk 1 above)
Comment (kusid) [reply mkdocs/docs/admin-guide.md]: right, I should've actually put the semicolon before 'therefore'

**Pull request merged**